client/src/components/Form.js

```diff
@@ -1,8 +1,11 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import Buzzwords from './Buzzwords';
 import isValidMentor from '../lib/validateFunctions';
+import loadFromLocal from '../lib/loadFromLocal';
+import saveToLocal from '../lib/saveToLocal';
+import {isValidMentorName, isValidAbout, isValidBuzzwords, isValidCompetence, isValidEmail, isValidPhone} from '../lib/validateFunctions'
```

Comment:
You can also add the default import from line 5 here. However do you even need all single functions here? I only found the `isValidMentorName` in the document
```suggestion
import isValidMentor, {isValidMentorName, isValidAbout, isValidBuzzwords, isValidCompetence, isValidEmail, isValidPhone} from '../lib/validateFunctions'
```